tests/EEGPSR.tex

```diff
@@ -1,129 +1,168 @@
+
+These \textit{unexpected} events will only occur within the field of view of the robot, and include:
+\begin{itemize}
+	\item \textbf{Waving:} The operator waves to the robot expecting the robot to approach to them, prior to give a command.
+	\item \textbf{Falling down:} The person being followed by the robot suddenly falls down, robot must offer assistance.
+	\item \textbf{Walking away:} One of the people in the crowd being counted suddenly leaves the room. \\ \textbf{Remark: in this case the people count reported by the robot will be valid for both: including and excluding the person who left.}
+	\item \textbf{Door shutting:} A door is suddenly shut while the robot is on its way to it.
+\end{itemize}
+
+\paragraph*{Remark:} for security concerns, team must notify the referees when the robot lacks of awareness.
+
+\subsection{Additional rules and remarks}
+\label{sec:eegpsr-remarks}
+\begin{enumerate}
+	\item \textbf{CONTINUE rule:} Teams are able to use the CONTINUE rule in this test, with all the standard penalties it involves as described in section \refsec{asrcontinue}.
+	%The CONTINUE rule can only be used with the custom operator (e.g. both penalties of custom speaker and CONTINUE rule will be applied). \\
+
+	\item \textbf{Number of Teams and Scheduling:} In each test slot, \eegpsrTeams teams will be competing in the arena concurrently. The robots will be tested in an interleaved fashion: The robots will retrieve commands and execute the task one after the other. As stated above, each robot will have a maximum amount of \eegpsrMaxCmdTime minutes per command (including time for retrieving the command and executing it). However, the sum of each of the times it took to carry each command should not exceed \eegpsrMaxTeamTime minutes. This means that if the robot took
+%	4 minutes to carry out its first command and another 4 minutes for its second command, it only has 2 minutes to carry out its third command. However, if it only took
+	2 minutes for the first and 2 minutes for the second, it still only has \eegpsrMaxCmdTime minutes for the third. To facilitate a fluent and untroubled performance of the robots, the other robots must always return or being returned to their designated position before the next robot will be given a command. If a robot moves from its designated position while another robot is working on a command, it must be immediately disabled and moved to its designated position. If a restart is still available to the team, it can be restarted at its designated position. To carry the robot, at most two team members are allowed in the arena, and the robot must be moved as quickly as possible. To start or restart the robot, at most one team member may operate the robot. The team members moving and operating the robots must leave the arena immediately after the robot is placed or started. \\

```

Comment:
I checked it twice and still agree. If robot A is working on a command and robot B starts moving... well, it shouldn't happen since it's not its turn. Disabling B sounds good for security reasons and to avoid interference.